lib/classes/CLI.js

```diff
@@ -60,23 +60,23 @@ class CLI {
     const commands = processedInput.commands;
     const options = processedInput.options;
 
-    // if only "help" or "h" was entered
-    if ((commands.length === 0 && (options.help || options.h)) ||
-      (commands.length === 1 && (commands.indexOf('help') > -1))) {
-      this.asciiGreeting();
-      this.generateMainHelp();
-      return true;
-    }
     // if only "version" or "v" was entered
     if ((commands.length === 0 && (options.version || options.v)) ||
-      (commands.length === 1 && (commands.indexOf('version') > -1))) {
-      this.asciiGreeting();
+        (commands.length === 1 && (commands.indexOf('version') > -1))) {
       this.getVersionNumber();
       return true;
     }
+
+    // if only "help" or "h" was entered
+    if ((commands.length === 0) ||
```

Comment:
If this line will be checked first we'll never get to check for the version option (or other options). I thought it will be a cleaner way to do that without adding more checks to this condition.